finagle-zipkin-core/src/main/scala/com/twitter/finagle/zipkin/core/Annotation.scala

```diff
@@ -1,4 +1,4 @@
-package com.twitter.finagle.zipkin.thrift
+package com.twitter.finagle.zipkin.core
```

Comment:
In the breaking section I listed the public classes (and companion objects) affected.:

>   * finagle-zipkin: Moved case classes `Span`, `ZipkinAnnotation`,
>     `BinaryAnnotation`, `Endpoint` and `Sampler` to finagle-zipkin-core.

https://github.com/sveinnfannar/finagle/blob/a7fbf5370dffd88a0d054b9e45ca2ca4b6ee3592/CHANGES#L57..L59